Decred Wallet/Features/Security/PasswordSetupViewController.swift

```diff
@@ -34,32 +50,49 @@ class PasswordSetupViewController: SecurityBaseViewController, UITextFieldDelega
         self.tfPassword.delegate = self
         self.tfConfirmPassword.delegate = self
         
-        self.headerText.text = String(format: LocalizedStrings.createPassword, self.securityFor)
-        self.lbMatchIndicator.text = " " // use empty space so label height isn't reduced
     }
     
     @objc func passwordTextFieldChange() {
+        
         let passwordStrength = PinPasswordStrength.percentageStrength(of: self.tfPassword.text ?? "")
         pbPasswordStrength.progress = passwordStrength.strength
         pbPasswordStrength.progressTintColor = passwordStrength.color
         
+        if self.tfPassword.text == ""{
+            self.passwordCount.text = "\(0)"
+        }else{
+            self.passwordCount.text = "\(self.tfPassword.text!.count)"
+        }
+        
         self.checkPasswordMatch()
     }
     
+
```

Comment:
Single empty line space is usually sufficient. Delete this and all other instances of multiple empty line spaces.